tests/test_units/test_table.py

```diff
@@ -8,25 +8,50 @@
 from fmu.dataio._objectdata_provider import _ObjectDataProvider
 
 
-def assert_works(file_path, answer):
-    """does the assert work for all tests
-
+def read_dict(file_path):
+    """Reads text file into dictionary
     Args:
```

Comment:
Are these functions pure helper functions? Could (or should) they be defined at pytest fixtures? If not; perhaps adding a front underscore ala `_read_dict()` would be beneficial?